python/desc/imsim/imSim.py

```diff
@@ -439,11 +451,8 @@ def sources_from_list(object_lines, obs_md, phot_params, file_name,
                            np.logical_and(ypix>y_min-pix_tol,
                                           ypix<y_max+pix_tol)))))
```

Comment:
FWIW, this is equivalent to the more legible
```
on_chip = np.where( (mag_norm < max_mag) |
                    ( (xpix > x_min - pix_tol) & (xpix < x_max + pix_tol) &
                      (ypix > y_min - pix_tol) & (ypix < y_max + pix_tol) )
                  )
 ```

Reply:
OK thank you. I

n anycase, I didn't actually find any problems with the code.  So, it looks good to me.